webrtc/private-to-private.md

```diff
@@ -0,0 +1,121 @@
+# WebRTC Private-to-Private
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Working Draft | Active | r0, 2023-03-03  |
+
+Authors: [@mxinden]
+
+## Motivation
+
+libp2p transport protocol enabling two private nodes (e.g. two browsers) to establish a direct connection.
```

Comment:
Sorry for dumb question, but which protocol handles the "public-to-private" case? With TCP/Quic it's the "dctur dial-back", is there a similar mechanism here, or will it be handled by this "private-to-private"?

Reply:
For now this specification does not explicitly address the public-to-private case. The public node might as well act as a private node, following the private-to-private flow. I don't know whether we need an explicit connection reversal for the public-to-private case long term.

Reply:
Yeah I don't think either we need explicit connection reversal, since ICE will do it for us

So `private-to-private` isn't totally accurate either :cold_sweat: It's more like `any-to-private`
But can't bikeshed on the name forever